derive_builder_core/src/macro_options/darling_opts.rs

```diff
@@ -130,6 +130,7 @@ pub struct FieldLevelSetter {
     strip_option: Option<bool>,
     skip: Option<bool>,
     custom: Option<bool>,
+    each: Option<Ident>,
```

Comment:
Related to the comment about `skip`, it looks like `each` is _not_ being considered in [this check](https://github.com/colin-kiegel/rust-derive-builder/pull/199/files#diff-fce05abe2a1a48a344afbebd38b64914fde1d4131faa01fa9d32e5804b6f31eeR151) for `field_enabled`. That seems like an oversight.

Reply:
Fixed. And corresponding test is added.